jenkins/jobs/bmo_e2e_tests.pipeline

```diff
@@ -36,6 +36,7 @@ pipeline {
                   extensions: [[$class: "WipeWorkspace"],
                   [$class: "CleanCheckout"],
                   [$class: "CleanBeforeCheckout"],
+                  [$class: "PreBuildMerge", options: [mergeTarget: "main"]],
```

Comment:
btw isn't this the same pipeline used for older release branches as well ? In that case, this is not correct, we need to identify the base branch. 
/hold

Reply:
Good point! We actually have e2e tests for release-0.5. I'll update